### ISSUE TITLE
m9sweeper encountered an error while attempting to validate this deployment. If you would like to set m9sweeper to not reject images when an internal error occurs, please set the ADMISSION_CONTROLLER_DEFAULT_ACTION environment variable to allow.

### ISSUE THREAD

**Issue by Navnitkumarbarnwal** [m9sweeper/m9sweeper#304]
We are getting this error in while doing our deployments though we have not configured any such rule in m9sweeper , or can i get any config by which we can disable this

**Comment by jacobbeasley** [comment]
Yes, you can modify your validating webhook configuration to disable this feature for now. 

https://github.com/m9sweeper/m9sweeper/blob/main/manifests/charts/dash/templates/validatingwebhookconfiguration.yaml

**Comment by jacobbeasley** [comment]
You could check dash logs to determine why it encountered an error. Likely something is misconfigured, but without logs I can't know what. In either case, you can turn off the validating webhook (see above) to bypass this.
